src/components/ui-component/templates/form.css

```diff
@@ -0,0 +1,102 @@
+.nr-dashboard-form {
+	display: inline-block;
+	width: 100%;
+	overflow: auto;
+}
+.nr-dashboard-form form{
+ overflow: auto;
+ padding: 15px;
+}
+.nr-dashboard-form md-input-container {
+	width: 100%;
+	height: 100%;
+	padding: 10px;
+  margin: 0px;
+	display: inline-block;
+}
+.nr-dashboard-form md-datepicker {
+	padding-bottom: 10px;
+}
+.nr-dashboard-form md-switch {
+	margin-top: 0px;
+}
+
+.nr-dashboard-form input[type="checkbox"]{
+	height :auto !important;
+}
+
+.nr-dashboard-form-button {
+	width:50%;
+	float:left;
+  background-color:  #0094ce !important;
+	color: rgba(255, 255, 255, 0.87) !important;
+}
+.nr-dashboard-theme-dark  .nr-dashboard-form-button {
```

Comment:
All the colour css parts should reside in the two main theme files - dark.css and light.css.